### DISCUSSION TITLE
Can't login to Ubuntu

### DISCUSSION THREAD

**Discussion by allawitte** [utmapp/UTM#5998] [Linux]
I installed a new Ubuntu 22.0 (QEMU 7.2 Virtual machine) on UTM. When I start Ubuntu in the first time, it demands from me a password, what I did not set up. It was no any interface to set up a password. And I can not log in to the system.

**Comment by hanniabu** [comment]
Same. Were you able to figure this out?

**Comment by its-all-waves** [comment]
I had this problem with the Debian image provided by UTM. After install, there was an existing user named 'Debian'. The password ended up being 'debian' or 'Debian' as well.

**Reply by dixonge** [reply]
In the Ubuntu 22.04 QEMU it is similar
User: Ubuntu
Password: ubuntu

**Comment by FlashBarryAllen** [comment]
In the Ubuntu 20.04 QEMU it is similar
```
#./qemu-system-x86_64 --version
QEMU emulator version 6.1.0 (v2.6.0-46506-g74d70f8008)
```
ubuntu login: root
Password: 

Login incorrect
ubuntu login: ubuntu
Login timed out after 60 seconds.

Ubuntu 20.04.1 LTS ubuntu ttyS0

**Reply by FlashBarryAllen** [reply]
I fixed it with ubuntu 22.04. You can try it.

**Reply by dgm37** [reply]
Use sudo -s, user ubuntu is in wheel.

**Comment by Suntany** [comment]
I used identical name, server's name and nickname during ubuntu installation. It maybe helpful if those were set.
